tests/EdFi.Ods.Api.IntegrationTests/EducationOrganizationTestDataBuilder.cs

```diff
@@ -0,0 +1,233 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Data;
+using System.Text;
+using EdFi.Ods.Entities.NHibernate.CommunityOrganizationAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.CommunityProviderAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.EducationServiceCenterAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.LocalEducationAgencyAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.OrganizationDepartmentAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.SchoolAggregate.EdFi;
+using EdFi.Ods.Entities.NHibernate.StateEducationAgencyAggregate.EdFi;
+
+namespace EdFi.Ods.Api.IntegrationTests
+{
+    public class EducationOrganizationTestDataBuilder
+    {
+        private IDbConnection Connection { get; set; }
+        private readonly StringBuilder _sql = new StringBuilder();
+        public int TestLocalEducationAgencyCategoryDescriptorId { get; private set; }
+        public int TestProviderStatusDescriptorId { get; private set; }
+        public int TestProviderCategoryDescriptorId { get; private set; }
+
+        private EducationOrganizationTestDataBuilder()
+        {
+        }
```

Comment:
Is this needed?

Reply:
Yes, this prevents the class from being created except from the Initialize factory method.  Since we needed to do some database hits to get default test data, I went with a factory method pattern instead of just constructing it, to keep the constructor clean.